internal/example/resource.go

```diff
@@ -118,7 +117,7 @@ func makeHTTPListener(listenerName string, route string) *listener.Listener {
 			},
 		},
 		HttpFilters: []*hcm.HttpFilter{{
-			Name:       wellknown.Router,
+			Name:       "http-router",
```

Comment:
I'm overall fine with it, but is it on purpose not matching the previous content?